robottelo/test.py

```diff
@@ -391,15 +383,23 @@ def _get_vm_list(cls):
         vm_list_string = conf.properties[
             'performance.test.virtual_machines_list']
         cls.vm_list = vm_list_string.split(',')
-        cls.logger.debug('VM list: {}'.format(cls.vm_list))
 
     @classmethod
     def _set_testcase_parameters(cls, savepoint_name,
-                                 raw_file_path, stat_file_path):
+                                 raw_file_name, stat_file_name):
         # note: set savepoint empty to continue test without restore
         cls.savepoint = conf.properties.get(savepoint_name, '')
-        cls.raw_file_name = conf.properties[raw_file_path]
-        cls.stat_file_name = conf.properties[stat_file_path]
+        cls.raw_file_name = raw_file_name
+        cls.stat_file_name = stat_file_name
+
+    @classmethod
+    def _get_organization_id(cls):
+        """Get organization id"""
+        result = OrgCli.list(per_page=False)
+        if result.return_code != 0:
+            cls.logger.error('Fail to get organization id.')
```

Comment:
Sure. I think @elyezer has previously mentioned this concept to me. But I found out that your codes above: (https://github.com/danuzclaudes/robottelo/blob/perf-constants/robottelo/test.py#L152) has just used the `cls.logger` way, I just imitated what your team has done.